tests/misc/test_split.py

```diff
@@ -103,37 +103,39 @@
 áéíóú/áéíóú/áéíóú/
 """
 
+test_data_lines = test_data.strip().splitlines()
 
-class SplitTests(unittest.TestCase):
+
+class TestSplit:
 
     """Test split."""
 
-    def test_split(self):
+    @pytest.mark.parametrize('cmd, out',
+                             [case.split('/')[:-2]
+                              for case in test_data_lines])
```

Comment:
What's your opinion on having a local function for ``case.split('/')[:-2]`` that explains what it does? I am familiar with slicing but **Explicit is better than implicit.** :bowtie: